pkg/manifests/manifests.go

```diff
@@ -2,182 +2,92 @@ package manifests
 
 import (
 	"bytes"
-	"fmt"
 	"io"
-	"strings"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 
-	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
-
 	"k8s.io/apimachinery/pkg/util/yaml"
-
-	operatorv1 "github.com/openshift/api/operator/v1"
-	"github.com/openshift/cluster-dns-operator/pkg/operator"
 )
 
 const (
-	ClusterDNSDefaultCR = "assets/cluster-dns-cr.yaml"
-
-	DNSNamespace          = "assets/dns/namespace.yaml"
-	DNSServiceAccount     = "assets/dns/service-account.yaml"
-	DNSClusterRole        = "assets/dns/cluster-role.yaml"
-	DNSClusterRoleBinding = "assets/dns/cluster-role-binding.yaml"
-	DNSConfigMap          = "assets/dns/configmap.yaml"
-	DNSDaemonSet          = "assets/dns/daemonset.yaml"
-	DNSService            = "assets/dns/service.yaml"
+	DNSNamespaceAsset          = "assets/dns/namespace.yaml"
+	DNSServiceAccountAsset     = "assets/dns/service-account.yaml"
+	DNSClusterRoleAsset        = "assets/dns/cluster-role.yaml"
+	DNSClusterRoleBindingAsset = "assets/dns/cluster-role-binding.yaml"
+	DNSConfigMapAsset          = "assets/dns/configmap.yaml"
+	DNSDaemonSetAsset          = "assets/dns/daemonset.yaml"
+	DNSServiceAsset            = "assets/dns/service.yaml"
+
+	// OwningClusterDNSLabel should be applied to any objects "owned by" a
+	// cluster dns to aid in selection (especially in cases where an ownerref
+	// can't be established due to namespace boundaries).
+	OwningClusterDNSLabel = "dns.openshift.io/cluster-dns"
```

Comment:
This is an entirely new label, right? Can we use `operator.openshift.io/dns`?

Reply:
done